src/components/Select/select.scss

```diff
@@ -91,6 +91,78 @@
         }
     }
 
+    .contacto-multi-select {
+        border-radius: 0;
+        box-shadow: 0px 0px 8px rgba(106, 110, 128, 0.15);
+        border: 1px solid var(--border-color-1);
+        height: var(--height);
+        width: 100%;
+
+        &--no-shadow {
+            box-shadow: none;
+        }
+
+        .close-tag-icon {
```

Comment:
Is this style is related to Tag component? If yes, can we please check if we can move it to Tag.scss file?